packages/openapi-validator/package.json

```diff
@@ -37,6 +37,9 @@
     "dist"
   ],
   "dependencies": {
+    "@types/axios": "^0.14.0",
+    "@types/request": "^2.48.7",
+    "axios": "^0.21.4",
```

Comment:
Thanks, I accidentally didn't include `axios` in `openapi-validator`'s package.json 🙂 I guess it doesn't break anything because we use it only [for types](https://github.com/openapi-library/OpenAPIValidators/blob/d5e520540d555389cf490892e95b5dccca37c9ce/packages/openapi-validator/lib/classes/AxiosResponse.ts#L1-L1).

Since we only use it for types, it should be in `devDependencies`. Same for `@types/request`. 

Also `axios` already has the types, so `@types/axios` [is deprecated](https://www.npmjs.com/package/@types/axios).